src/main/java/online/book/store/model/CartItem.java

```diff
@@ -0,0 +1,43 @@
+package online.book.store.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Data
+@Entity
+@SQLDelete(sql = "UPDATE cart_items SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```

Comment:
It's up to you, but you can use `@SQLRestriction("is_deleted=false")` instead of deprecated `@Where`

Reply:
agree :)

Reply:
> agree :)

Yes, I added this to the list of fixing commit. Because I used @Where everywhere in my code